packages/contracts/test/registry/plugin-repo-registry.ts

```diff
@@ -157,4 +157,86 @@ describe('PluginRepoRegistry', function () {
       .to.be.revertedWithCustomError(ensSubdomainRegistrar, 'AlreadyRegistered')
       .withArgs(pluginRepoNameDomainHash, ensSubdomainRegistrar.address);
   });
+
+  // without mocking we have to repeat the tests here to make sure the validation is correct
+  describe('name validation', () => {
+    it('should validate the passed name correctly (< 32 bytes long name)', async () => {
+      const baseName = 'this-is-my-super-valid-name';
+
+      // loop through the ascii table
+      for (let i = 0; i < 127; i++) {
```

Comment:
can't we have this tests as a common test which can be inserted directly in the dao-registry + plugin-repo-registry ? To avoid `if ((i > 47 && i < 58) || (i > 96 && i < 123) || i === 45) {` in 3 places.

Maybe that test would expect the the subdomainName + some tx which can be used in the for loop.